blockchain/staking/run_a_validator.md

```diff
@@ -44,6 +56,12 @@ Session keys for your node: [long text starting with 0x here]
 
 Copy the session keys, you will need them when adding validator.
 
+To get session keys manually, run the command and copy the 'result' parameter from the result of command.
```

Comment:
Consider just linking to `[Regenerating session keys](./regenerate_session_keys.md)` .